search/utils/constants.py

```diff
@@ -1,3 +1,5 @@
+# These should really belong in settings.py
```

Comment:
@dhilipsiva  ok :+1: 
But I felt during development and while creating big projects we usually have lot of constants that we keep using. So if we keep all of them in settings.py file it will make it big and not easy to manage when we need to reference and change it quickly over period of time. So excluded it from settings.py and placed it separately.